scenarios/happypath/build/build_application.go

```diff
@@ -17,20 +19,24 @@ func Application(env env.Env) (bool, error) {
 	if !ok {
 		return false, nil
 	}
+	log.Infof("First job was triggered")
 
 	// Get job
 	ok, jobSummary := test.WaitForCheckFunc(env, isJobListed)
 	if ok {
```

Comment:
Should use idiomatic go error handling instead with if !ok { exit } to reduce cyclomatic complexity.

Reply:
Changes applied.